### ISSUE TITLE
2019-11-29 tc017 M4SD executes but doesn't generate estimated concentration results

### ISSUE THREAD

**Issue by tensfeldt** [tensfeldt/openNCA#70]
This occurs without or without partial AURCs being defined in the MCT. An empty estimated concentration dataset, i.e. just headers, is generated.

This appears to be new to the current build 22d43ea

**Comment by rudraya** [comment]
This issue was addressed in 6e99fab, please let me know if it still occurs.

**Comment by tensfeldt** [comment]
With commit 6e99fab, the estimated concentration dataset is now being generated/returned. This issue can be closed.

```
> mct[, names(parameter_required("^AUC.[0-9]+?.", names(mct), simplify=FALSE))]
  AUC.1.T1 AUC.1.T2 AUC.2.T1 AUC.2.T2
1        2       18        8       16
> e
    PKDATAROWID     SDEID TIME   CEST_KEL  CEST_INT CEST_EXT CEST_C0 CEST_TLAST
1      22321820  58098661    4 42712.2304        NA       NA      NA         NA
2      22321821  58098661   12 15595.0137        NA       NA      NA         NA
4            NA  58098661   16         NA 1961454.9       NA      NA         NA
3      22321822  58098661   24  3440.6146        NA       NA      NA         NA
11           NA  58613088   16         NA 1676466.5       NA      NA         NA
12     22321905 107790547    4 36106.3730        NA       NA      NA         NA
21     22321906 107790547   12 16426.1381        NA       NA      NA         NA
41           NA 107790547   16         NA 1942784.0       NA      NA         NA
31     22321907 107790547   24  5040.3753        NA       NA      NA         NA

```
